app/lib/constants/site_config.rb

```diff
@@ -34,7 +34,7 @@ module SiteConfig
         placeholder: "Campaign sidebar enabled or not"
       },
       campaign_sidebar_image: {
-        description: "https://image.url",
+        description: "https://image.url/image.png",
```

Comment:
Rather than make these changes everywhere in the SiteConfig, we could assign `"https://image.url/image.png"` to a constant that we `freeze` in this file.

That way, if we decide to move from `image.png` to `image.jpg` down the road (for example), instead of changing every single one of these instances yet again, you need only change the constant. It also ensures that a contributor down the road can easily use the same constant again (instead of manually typing it out and then potentially making a typo or even accidentally specifying the wrong image type).

Reply:
I appreciate the suggestion! 

I attempted to add an `IMAGE_PLACEHOLDER` constant, but was unable to get it to work. After trying a couple of different approaches, I dug into it a little more and realized that the file itself is a giant constant, which was causing the constant that I tried to add, to not render properly in the SiteConfig. I'm guessing that this is also why we haven't assigned any of the placeholder text in this file to a constant, but 🤷‍♀️  @vaidehijoshi

Reply:
@juliannatetreault The file _itself_ isn't a constant, it's a module. However, there is a hash inside of this module that encapsulates the entire `DETAILS` of the SiteConfig (see [this line](https://github.com/forem/forem/blob/master/app/lib/constants/site_config.rb#L3)). I suspect you are probably assigning the object in the wrong place/incorrectly.

I created a `TEST` constant locally above the `DETAILS` object:
```ruby
TEST = "i love llamas".freeze
DETAILS = {
  # ...
}
```

And then used it inside of SiteConfig:
```ruby
campaign_sidebar_image: {
  description: TEST,
  placeholder: "Used at the top of the campaign sidebar"
},
```

and it does work:
<img width="982" alt="Screen Shot 2020-10-21 at 9 32 55 AM" src="https://user-images.githubusercontent.com/6921610/96750394-be09ff80-1380-11eb-8234-6298dc05eb31.png">

Reply:
Oh, my apologiess -- that works! Why did you not put it in an array, though?

Reply:
@juliannatetreault If it's just a single string that we're going to reuse, it doesn't need to be in an array, right?  Constants can be anything -- strings, numbers, arrays, hashes/objects! In this case, we're using the constant to represent a single piece of placeholder text, so it doesn't need to be in array I think.

Just so I can understand your thinking and help clarify here, let me ask a question in reply to your question 😄 -- why did you think it should be in an array?

Reply:
Interesting! I thought it needed to be in an array because I've only ever seen constants in an array... the more you know. 😅

Reply:
@juliannatetreault Heh, nope! In fact, the `DETAILS` constant isn't even an array -- it's a hash 😉 

Here's [a great resource](https://www.rubyguides.com/2017/07/ruby-constants/) on Ruby constants :)